spec/observables/using-spec.ts

```diff
@@ -1,24 +0,0 @@
-import * as Rx from '../../dist/cjs/Rx.KitchenSink';
-import {it} from '../helpers/test-helper';
-
-const Observable = Rx.Observable;
-const Subscription = Rx.Subscription;
-
-describe('Observable.using', () => {
-  it('should dispose of the resource when the subscription is disposed', (done) => {
-    let disposed = false;
-    const source = Observable.using(
-      () => new Subscription(() => disposed = true),
-      (resource) => Observable.range(0, 3)
-    )
-    .take(2);
-
-    source.subscribe();
-
-    if (disposed) {
-      done();
-    } else {
-      done.fail('disposed should be true but was false');
-    }
-  });
-});
```

Comment:
I'll check again, but when I was working on this, there was no `using` implementation for it to test in the library. It seemed to be a file that was accidentally merged.